examples/pipelines/datacomp/pipeline.py

```diff
@@ -51,20 +50,15 @@
         "spacy_pipeline": "en_core_web_sm",
         "batch_size": 1000,
         "min_complexity": 1,
-        "min_num_actions": 1,
-    },
-)
-cluster_image_embeddings_op = ComponentOp(
-    component_dir="components/cluster_image_embeddings",
-    arguments={
-        "sample_ratio": 0.3,
-        "num_clusters": 3,
+        "min_num_actions": 0,
     },
 )
 
 # add ops to pipeline
 pipeline.add_op(load_from_hub_op)
 pipeline.add_op(filter_image_resolution_op, dependencies=load_from_hub_op)
 pipeline.add_op(filter_complexity_op, dependencies=filter_image_resolution_op)
-pipeline.add_op(cluster_image_embeddings_op, dependencies=filter_complexity_op)
 # TODO add more ops
+
+
+client.compile_and_run(pipeline=pipeline)
```

Comment:
Can you put this in a `if __name__ == "__main__":` block so it's not executed when running `fondant run pipeline:pipeline`.